docs/source/building.rst

```diff
@@ -0,0 +1,313 @@
+.. index:: building
+
+********
+Building
+********
+
+
+Introduction
+============
+
+Bulebule is an open and freely distributed micromouse design. If you plan on
+starting your own micromouse, you may consider at least having a look at how it
+is built for reference!
+
+Most of the components are easy to find, easy to solder and cheap, with the
```

Comment:
I would reorder it : "with the exception of the motors which we found second-hand, the rest of the components are easy to find, easy to solder and cheap"

Reply:
10d3090780a448215bad04f05b6dafaf2897588e